src/lib-storage/index/dbox-multi/mdbox-save.c

```diff
@@ -25,6 +25,7 @@ struct dbox_save_mail {
 	uint32_t seq;
 	uint32_t append_offset;
 	bool written_to_disk;
+	time_t save_date;
```

Comment:
Could you move this on top of the bool? Practically won't matter now, but generally we keep bools at the end of structs.

Reply:
Done!